content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -94,7 +94,7 @@ A fair criticism of the Foundation is that we haven’t shared publicly about wh
 
 If you encounter harmful content on Matrix, you can report it in a few ways:
 
-* If you are a Matrix.org homeserver user, you can [report the content](https://github.com/element-hq/element-x-ios/pull/587) from your client, and it will head to our Safety team. Matrix clients like Element iOS now allow users to [report rooms](https://github.com/element-hq/element-ios/pull/7805), and we understand it will be available in Element X, Element Android and Element Web soon.  
+* If you are a Matrix.org homeserver user, you can [report the content](https://github.com/element-hq/element-x-ios/pull/587) from your client, and it will head to our Trust & Safety team. Matrix clients like Element iOS now allow users to [report rooms](https://github.com/element-hq/element-ios/pull/7805), and we understand it will be available in Element X, Element Android and Element Web soon.  
```

Comment:
I would ideally want to add more ecosystem clients here if there are any that support it https://matrix.org/ecosystem/clients/ but I don't know if/which do at this point